content/manuals/subscription/manage-seats.md

```diff
@@ -0,0 +1,112 @@
+---
+description: Learn how to add or remove seats for an existing subscription
+keywords: Docker, Docker Hub, subscription, update, add, seats, pricing
+title: Manage subscription seats
+linkTitle: Manage seats
+aliases:
+- /docker-hub/billing/add-seats/
+- /subscription/add-seats/
+- /docker-hub/billing/remove-seats/
+- /subscription/remove-seats/
+- /subscription/core-subscription/add-seats/
+- /subscription/core-subscription/remove-seats/
+weight: 20
+---
+
+You can add seats at anytime to your existing subscription.
+
+When you add seats to your subscription in the middle of your billing cycle, you are charged a prorated amount for the additional seats.
+
+{{< include "tax-compliance.md" >}}
+
+## Add seats
+
+{{< tabs >}}
+{{< tab name="Docker subscription plan" >}}
+
+> [!IMPORTANT]
+>
+> If you have a [sales-assisted Docker Business subscription](details.md#sales-assisted), contact your account manager to add seats to your subscription.
+
+To add seats to your subscription:
+
+1. Sign in to the [Docker Billing Console](https://app.docker.com/billing/).
+2. Select your account from the drop-down menu in the top-left.
+3. Select **Add seats**.
+4. Follow the on-screen instructions to complete adding seats.
```

Comment:
**nitpick:** It would be handy to break this down into specific steps so that the customer can read what exactly to do and where to click.